examples/platform/efr32/EFR32DeviceAttestationCreds.cpp

```diff
@@ -74,8 +74,8 @@ class DeviceAttestationCredsEFR32 : public DeviceAttestationCredentialsProvider
         uint8_t signature[64] = { 0 };
         size_t signature_size = sizeof(signature);
 
-        psa_status_t err = psa_sign_hash(key_id, PSA_ALG_ECDSA(PSA_ALG_SHA_256), digest_to_sign.data(), digest_to_sign.size(),
-                                         signature, signature_size, &signature_size);
+        psa_status_t err = psa_sign_message(key_id, PSA_ALG_ECDSA(PSA_ALG_SHA_256), digest_to_sign.data(), digest_to_sign.size(),
```

Comment:
@rcasallas-silabs It's really confusing to have the argument be called `digest_to_sign` if it's not actually a digest.  Please rename it as needed?

Reply:
Certainly. Fixed with [#20686](https://github.com/project-chip/connectedhomeip/pull/20686